### DISCUSSION TITLE
Installing the Debian shared library package also installs a web browser

### DISCUSSION THREAD

**Discussion by lovell** [libvips/libvips#2812] [General]
I've been made aware of the following tweet:

https://twitter.com/bketelsen/status/1526728256736616448

> I love* that when I install libvips on Debian it also brings in Firefox as a dependency. That makes perfect sense.
>
> *do not love

It looks like the `libvips42` shared library package in Debian has a "rec(ommended)" dependency on `nip2`.

https://packages.debian.org/bullseye/libvips42

Should we request that this be lowered to "sug(gested)" or removed entirely? (The `libvips-tools` package already has a "rec" dependency on `nip2`, which makes sense.)

**Comment by jcupitt** [comment]
Debian has always done this, it's frustrating. `apt install libvips42` is almost 800mb, `apt install libvips42 --no-install-recommends` is 28mb.

I've asked a couple of times about removing the nip2 recommendation, but without success. Perhaps if you ask as well?

**Comment by lovell** [comment]
Here's the line that I think would need removing:

https://sources.debian.org/src/vips/8.12.2-1/debian/control/#L26
```diff
- Recommends: nip2
```

Would the use of Debian's email-based `reportbug` tooling to create a "bug" be the right approach, so at least there's something to track?

**Reply by jcupitt** [reply]
I've tried mailing the maintainer directly before and that failed horribly, so yes, probably.

**Comment by kleisauke** [comment]
This has been fixed in Debian Unstable, see:
https://sources.debian.org/src/vips/8.14.1-3/debian/control/#L25
http://launchpadlibrarian.net/651152053/vips_8.14.1-1build1_8.14.1-2.diff.gz
```diff
@@ -23,7 +22,7 @@
 Pre-Depends: ${misc:Pre-Depends}
 Depends: ${misc:Depends}, ${shlibs:Depends}
 Provides: libvips
-Recommends: nip2
+Suggests: nip2
 Description: image processing system good for very large ones
  VIPS is an image processing system designed with efficiency in mind.
  It is good with large images (ones that larger than the amount of RAM in
@@ -71,8 +70,7 @@
 Section: graphics
 Architecture: any
 Depends: ${misc:Depends}, ${shlibs:Depends}, python3
-Recommends: nip2, libvips-doc
-Suggests: libvips-dev
+Suggests: nip2, libvips-doc, libvips-dev
 Description: image processing system good for very large ones (tools)
  VIPS is an image processing system designed with efficiency in mind.
  It is good with large images (ones that larger than the amount of RAM in
```

And the corresponding Changelog note:
https://tracker.debian.org/media/packages/v/vips/changelog-8.14.1-3
>   * Suggest only nip2 on libvips42 and libvips-tools.
